types/braintree-web/modules/apple-pay.d.ts

```diff
@@ -64,10 +64,32 @@ export enum ApplePayStatusCodes {
     STATUS_PIN_LOCKOUT,
 }
 
+export type ApplePayTokenizeValues = 'Yes' | 'No' | 'Unknown';
+
+export interface ApplePayDetails {
+    cardType: string;
+    cardholderName: string;
+    dpanLastTwo: string;
+    paymentInstrumentName: boolean;
```

Comment:
I didn't see `paymentInstrumentName` in the docs. Is it undocumented?

Reply:
yep, I took it from real response, not from docs... I'll remove it, so it would match with the doc